src/pageText.js

```diff
@@ -170,54 +169,57 @@ async function addListener(listener, {
                 }
             }
         },
-        {
-            type: "webScience.pageText.parsedText",
-            schema: {
-                pageId: "string",
-                url: "string",
-                title: "string",
-                content: "string",
-                textContent: "string",
-                privateWindow: "boolean"
-            }
-        });
+            {
+                type: "webScience.pageText.parsedText",
+                schema: {
+                    pageId: "string",
+                    url: "string",
+                    title: "string",
+                    content: "string",
+                    textContent: "string",
+                    privateWindow: "boolean"
+                }
+            });
 
         // Notify the content script when there is a new Readability status
         // for a page and the page URL matches at least one listener
         messaging.registerSchema("webScience.pageText.isArticle", {
             isArticle: "boolean"
         });
-        browser.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
-            if("isArticle" in changeInfo && "url" in tab) {
+        browser.tabs.onUpdated.addListener((tabId, _changeInfo, tab) => {
+            if ("url" in tab) {
                 // Test match patterns here rather than in the tabs.onUpdated
                 // listener options so we don't have to manage multiple listeners
                 // or remove and add the listener while events might be queued
                 for (const listenerRecord of textParsedListeners.values()) {
                     if (listenerRecord.matchPatternSet.matches(tab.url)) {
                         messaging.sendMessageToTab(tabId, {
                             type: "webScience.pageText.isArticle",
-                            isArticle: tab.isArticle
+                            isArticle: true
```

Comment:
hmm this feels confusing. I guess I'm not really sure why it was like this in the first place. really the schema should remove this boolean since we're not checking for article-ness here anymore, right? and the explanation comments in `pageText.content.js` should be updated to reflect the new logic.

I just don't want our (currently good) code legibility to indicate one thing, but then actually things work differently than you expect

Reply:
These are the comments I'm referring to:

https://github.com/mozilla-rally/web-science/blob/8fe8c055b6c87f6240dbba299243a356950bd688/src/content-scripts/pageText.content.js#L36
https://github.com/mozilla-rally/web-science/blob/8fe8c055b6c87f6240dbba299243a356950bd688/src/content-scripts/pageText.content.js#L38

Reply:
Yeah I'm a little on the fence, checking for readability like this is a Firefox only feature in the WebExtension API, we could continue to support it only for Firefox I suppose, but I am guessing it's a really marginal win and makes the code a lot more confusing.

I'll modify the comments, thanks for pointing that out, and we could also just not send that boolean if we're not going to keep Firefox support.

Reply:
I didn't have time to dive into the source code, but I'm preeeetty certain that the isArticle Firefox feature just uses the isProbablyReaderable function under the hood. So the only regression for Firefox is that it has to basically do the check twice, I suppose?